stablehlo/reference/Ops.h

```diff
@@ -44,6 +45,12 @@ Tensor evalTransposeOp(Type resultType, const Tensor &operand,
                        DenseIntElementsAttr permutation);
 Tensor evalXorOp(const Tensor &lhs, const Tensor &rhs, Type resultType);
 
+// The eval functions for the following ops are used only for test harness.
+llvm::Error evalExpectAlmostEqConstOp(const Tensor &lhs, ElementsAttr value);
+llvm::Error evalExpectAlmostEqOp(const Tensor &lhs, const Tensor &rhs);
+llvm::Error evalExpectEqConstOp(const Tensor &lhs, ElementsAttr value);
+llvm::Error evalExpectEqOp(const Tensor &lhs, const Tensor &rhs);
```

Comment:
Can we move these functions into a separate file in `tests/`?

Reply:
done.